catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/TreUtility.java

```diff
@@ -73,4 +79,42 @@ public static Integer convertToInteger(Tre tre, String fieldName) {
         }
         return Optional.empty();
     }
+
+    public static Float convertToFloat(Tre tre, String fieldName) {
```

Comment:
We should either return `Optional<Float>` or add `@Nullable` on the return types.

Same comment for all the other conversion methods.